src/core/storage/btree/Btree.cpp

```diff
@@ -265,7 +273,7 @@ TEST_CASE("Btree persistence", "[btree]") {
 	fs::create_directories("/tmp/eugene-tests/btree-persistence");
 
 	SECTION("Loading and recovery") {
-		std::map<typename DefaultConfig::Key, typename DefaultConfig::Val> backup;
+		std::map<typename Config::Key, typename Config::Val> backup;
 		Position rootpos;
 		{
 			if constexpr (NDEBUG)
```

Comment:
I think all these checks are always correct. What is this NDEBUG?

Reply:
I want to be able to toggle debug prints. This is through the NDEBUG flag.